### PR TITLE
Aws rotate deploy keys

### PR DESCRIPTION
**Pull request by truesysadmin**
Closes #ISSUE_NUMBER

### CODE REVIEW

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -347,8 +347,8 @@ jobs:
 
     - name: Publish assets
       env:
-        AWS_ACCESS_KEY_ID: ${{ secrets.aws_user_circleci_rocketchat_access_key_id }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.aws_user_circleci_rocketchat_secret_access_key }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.aws_user_github_rocketchat_access_key_id }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.aws_user_github_rocketchat_secret_access_key }}
```
Comment (kapcod) [inline .github/workflows/build_and_test.yml:351]: In "Rotate  AWS key " job you use upper case, here lower case? I don't know if this is case sensitive or not, but please use the real case of the secret key.

Comment (truesysadmin) [reply .github/workflows/build_and_test.yml:351]: It is not case sensitive in meaning that it makes it upper case  when you work with  secrets. Already tested it with samw

Comment (truesysadmin) [reply .github/workflows/build_and_test.yml:351]: it is made as it was made before - just changed the name  of the variable.

File: .github/workflows/aws_key_rotate.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: 'Rotate  AWS key'
```
Comment (nmagedman) [inline .github/workflows/aws_key_rotate.yml:1]: Remove the extra space

File: .github/workflows/aws_key_rotate.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
```
Comment (nmagedman) [inline .github/workflows/aws_key_rotate.yml:6]: Add comment `# monthly`

File: .github/workflows/aws_key_rotate.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+name: 'Rotate  AWS key'
+
+
+on:
+  schedule:
+    - cron: '0 0 1 * *' 
+
+jobs:
+  rotate:
+    name: rotate iam user keys
```
Comment (nmagedman) [inline .github/workflows/aws_key_rotate.yml:10]: It's weird having all of:
* rotate iam user keys
* rotate aws keys
* rotate aws secrets

Let's use one name consistently:

```
jobs:
  rotate-aws-secrets:
    name: Rotate AWS secrets
```

Use the name both here and in the single step's name in line 13.

File: .github/workflows/aws_key_rotate.yml (hunk 3 above)
Comment (nmagedman) [reply .github/workflows/aws_key_rotate.yml:6]: and avoid trailing whitespace on the line!

**Review by nmagedman — COMMENTED** [review]
I've incorporated all of these aws_key_rotate.yml changes onto the scheduled-events branch,
and the build_and_test.yml change onto the latest (WIP) sa_patches_* branch.

Closing.

**Pull request closed without merging**